timed/tracking/serializers.py

```diff
@@ -41,7 +41,8 @@ def validate(self, data):
 
         # validate that there is only one active activity
         activity = models.Activity.objects.filter(user=user)
-        if activity.filter(to_time__isnull=True) and to_time is None:
+        if (activity.filter(to_time__isnull=True) and
```

Comment:
what happens when I patch activity and set `to_time` to `None`? I guess we could have more than one activity with `to_time` `None` right?

Reply:
I added an test and changed the condition

Reply:
Changed it to how we discussed it